### PR TITLE
Show notifications when generator starts / stops

### LINKED ISSUE #1475: UI changes for connected gensets
as per gui-v1 commit b9583fd7bee8f1120402ba4697693765c6f06f26

I think most of these are done, except for the text notifications

    UI changes for connected gensets
    https://github.com/victronenergy/venus/issues/1288
    
    - Remove "Mode" option
    - Move "Autostart" and "Manual start" from the autostart menu to the toplevel
    - Show GX control (start/stop service) status and error code
    - Show genset "Remote start mode" status
    - Add text notification and disable the manual control menu items
    when "remote start mode" on genset is disabled.
    
    The generator start/stop menu from the main settings menu is unchanged.

### PR DESCRIPTION
**Pull request by DanielMcInnes**
as per gui-v1 commit b9583fd7bee8f1120402ba4697693765c6f06f26 Fixes #1475

### CODE REVIEW

File: components/dialogs/GeneratorStartDialog.qml [hunk 1]
```diff
@@ -23,6 +23,13 @@ GeneratorDialog {
 	}
 
 	runGeneratorAction: function() {
+		Global.showToastNotification(VenusOS.Notification_Info,
+									 timedRunSwitch.checked && (timeSelector.hour || timeSelector.minute)
+									   //% "Starting. The generator will stop in %1, unless other conditions keep it running"
+									 ? qsTrId("generator_start_dialog_starting_will_stop").arg(Utils.formatHoursMinutes(timeSelector.hour, timeSelector.minute))
+									   //% "Starting, generator won't stop till user intervention"
+									 : qsTrId("generator_start_dialog_starting_wont_stop"),
+									 5000)
```
[the remark this answers is not in the document]
Comment (DanielMcInnes) [reply components/dialogs/GeneratorStartDialog.qml]: Thanks Bea, updated as suggested

**Review by blammit — APPROVED** [review]
Just minor comments re. indentation, LGTM.

File: components/PageGensetModel.qml [hunk 2]
```diff
@@ -386,6 +378,14 @@ ObjectModel {
 		}
 	}
 
+	ListNavigationItem { // to test, use the 'gdh' simulation. Not visible with the 'gdf' simulation.
+		text: CommonWords.settings
+			onClicked: {
```
[the remark this answers is not in the document]
Comment (DanielMcInnes) [reply components/PageGensetModel.qml]: fixed

File: components/dialogs/GeneratorStartDialog.qml [hunk 3]
```diff
@@ -65,10 +65,11 @@ GeneratorDialog {
 			wrapMode: Text.Wrap
 			color: Theme.color_font_primary
 			horizontalAlignment: Text.AlignHCenter
-			visible: root.generator.autoStart
-
-			//% "Generator will stop after the set time, unless autostart condition is met, in which case it will keep running."
-			text: qsTrId("controlcard_generator_startdialog_description")
+			text: timedRunSwitch.checked && (timeSelector.hour || timeSelector.minute)
```
[the remark this answers is not in the document]
Comment (DanielMcInnes) [reply components/dialogs/GeneratorStartDialog.qml:68]: done

**Pull request merged**